ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -244,14 +245,14 @@ public struct FluentNotification: View, TokenizedControlView {
                             RoundedRectangle(cornerRadius: tokenSet[.cornerRadius].float)
                                 .fill(Color(dynamicColor: tokenSet[.backgroundColor].dynamicColor))
                         )
-                        .shadow(color: Color(dynamicColor: tokenSet[.ambientShadowColor].dynamicColor),
-                                radius: tokenSet[.ambientShadowBlur].float,
-                                x: tokenSet[.ambientShadowOffsetX].float,
-                                y: tokenSet[.ambientShadowOffsetY].float)
-                        .shadow(color: Color(dynamicColor: tokenSet[.perimeterShadowColor].dynamicColor),
-                                radius: tokenSet[.perimeterShadowBlur].float,
-                                x: tokenSet[.perimeterShadowOffsetX].float,
-                                y: tokenSet[.perimeterShadowOffsetY].float)
+                        .shadow(color: Color(dynamicColor: shadowInfo.colorOne),
+                                radius: shadowInfo.blurOne,
+                                x: shadowInfo.xOne,
+                                y: shadowInfo.yOne)
+                        .shadow(color: Color(dynamicColor: shadowInfo.colorTwo),
+                                radius: shadowInfo.blurTwo,
+                                x: shadowInfo.xOne,
```

Comment:
Isn't it supposed to be `xTwo` and `yTwo` here?

Reply:
Yep good catch